src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -232,6 +244,14 @@
                 namespace,
                 (state: ADRState) => state.adrError
             ),
+            releaseName(){
+                const selectedDataset = this.selectedDataset
+                if (selectedDataset?.release && this.releases.length){
+                    return this.releases.find(release => release.id === selectedDataset.release)?.name || null
+                } else {
+                    return null
+                }
```

Comment:
The above won't compile in ts because it can return undefined due to the `?.name` and `this.selectedDataset.release` isn't guarded against the object being null.

Reply:
Sorry - I should have checked before spouting off!